### PR TITLE
Update standings page

### PR DESCRIPTION
**Pull request by mzachar6**

### CODE REVIEW

**Comment by mzachar6** [comment]
Hi,
I've used Card because it is suggested by Bootstrap as a replace for Panels
to: @buksy90

File: src/components/Tab.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React, { PureComponent } from 'react';
+
+export default class Tab extends PureComponent {
+
+  render() {
+    return (
+      <li className="nav-item">
+        <a className={`nav-link ${this.props.active} text-black` } href={`#${this.props.href}`} key={`${this.props.index}`} onClick={this.props.onClick} role="button" > {this.props.label} </a>
```
Comment (buksy90) [inline src/components/Tab.js]: We are not making any redirection, so lets replace "a" with "span" and remove href attribute

File: src/pages/TableList.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    }));
+  }
+
+  render() {
+
+    // Render List of tabs for team and players standing and higlight active one
+    const tabList = this.state.tabs.map((item, index) => {
+      let isActive = (this.state.active === index ? 'active bg-dark' : '');
+      return (
+        <Tab active={isActive} name={item} href={item.name} key={index} onClick={this.handleOnClick.bind(this, item, index)} label={item.label} />
```
Comment (buksy90) [inline src/pages/TableList.js]: I like that youre trying to divide the logic in small components ... but this time I think its over complicated. 
The Tab component is not really drawing any tabs, its just drawing navigation button (so the naming is not appropriate). And it is very case specific and probably not going to be reused.  I think you could have just render "raw" html this time.

File: src/pages/TableList.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import Tab from '../components/Tab'
+import StatsTable from '../components/StatsTable'
+
+export default class TableList extends Component {
+  constructor(props) {
+    super(props);
+
+    /** mock up table data  */
+    const teams = [
```
Comment (buksy90) [inline src/pages/TableList.js]: nice mocks for testing, we should start writing tests (I will set up some testing framework later). But mocks should not be part of production code, if you rebase your branch against master

git rebase master

you will be able to use new functions from data provider that return similar data

**Comment by mzachar6** [comment]
> Very good code in overall, I have few remarks but those require just small changes. I think I will prepare more difficult tasks for you soon ;)

Thank you. I appreciate it. Please review following changes
TableList Component commit:
• Mock-up data removed
• Link tag replaced with span tag
• Added component methods for fetching data from DataProvider
• <StatsTable> Component commented out

**Comment by buksy90** [comment]
hmm ... I checked gitk and the graph looks weird

1. every commit is doubled, take a look at "Adding TableList component link to main menu", very same commit is above the master and then again 6 commits above.

image

2. your branch has two "lines", looks like you have made a branch within your branch and then you merged them? not sure
image

Ideally, your branch should create new line from master, add few commits and then come back to master (after merge), just like docker branch did
image

3. when I switch to "Files changed" in this pull request, I can see in your request even files you didnt edited (players.php, composer.json, ...). This is probably due to bad rebase


I have fixed all git issues and created new branch issue_23 and also new Pull request for this branch. I copied all your commits to that branch and rebased it against master. I will comment your changes in this new pull request.

**Pull request closed without merging**